knative-kubecon/README.md

```diff
@@ -387,7 +390,69 @@ the *testevents* pod we've created earlier, which is the Source listening on Kub
 We can also roughly spot how the Knative Serving system works underneath, as our deployment gets requests from the activator and is connected
 to the autoscaler, to provide metrics there.
 
-## 4. Scaling up and down
+What we also spot: Our application has a problem. All requests from the dispatcher to the application are failing.
```

Comment:
ehm, I don't really get this, looking at the last(above) image from 3.

See: https://github.com/openshift-cloud-functions/demos/blob/ca996669772dd79d9b2f141d56f7ce4529e782b8/knative-kubecon/images/kiali.png

IMO the traffic is green, from dispatcher to the app -